packages/web/src/pages/audio-rewards-page/ChallengeRewardsTile.tsx

```diff
@@ -343,6 +399,12 @@ const RewardsTile = ({ className }: RewardsTileProps) => {
 
   const wm = useWithMobileStyle(styles.mobile)
 
+  const { isEmpty: shouldHideCumulativeRewards } = useChallengeCooldownSchedule(
+    {
```

Comment:
super nit, but can this all be consolidated into one line?

Reply:
linter makes it multiple lines